rocky-backend/DBtest.py

```
@@ -0,0 +1,95 @@
+import requests
+
+BASE = "http://127.0.0.1:5001"
+
+def test(name, condition):
+    print(f"[{'PASS' if condition else 'FAIL'}] {name}")
+
+print("\nDatabase Tests\n")
+
+# create valid user
+res = requests.post(f"{BASE}/users", json={
```

Ruff (0.15.7)

[error] Probable use of `requests` call without timeout

(S113)